boot/makebootable_test.go

```diff
@@ -270,7 +275,7 @@ version: 5.0
 	c.Check(files, HasLen, 1)
 	// check that the recovery bootloader configuration was copied with
 	// the correct content
```

Comment:
this comment is slightly wrong now, since we're not copying the recovery bootloader configuration, we are installing it, unless "copying" means writing from the snapd binary data file ? :smile: